tests/test_client.py

```diff
@@ -1198,66 +961,6 @@ def test_read_waveform_digital(tekhsi_client, header, response_data, expected_le
     assert len(waveform.y_axis_byte_values) == expected_length
 
 
-class DummyThread:
-    def __init__(self, should_raise=False):
-        self.should_raise = should_raise
-        self.join_called = False
-
-    def join(self, timeout=None):
-        self.join_called = True
-        if self.should_raise:
-            raise RuntimeError("Thread error")
-
-
-@pytest.mark.parametrize(
-    "connected, verbose, thread_active, should_raise, expected_output",
-    [
-        (False, False, False, False, ""),  # Not connected
-        (True, True, True, False, "close"),  # Verbose output
-        (True, False, True, False, ""),  # Thread handling without verbose
-    ],
-)
-def test_close(
```

Comment:
Why was this removed?

Reply:
I checked in the coverage tests, this was not covering the required functionality, though individual test was passing.